src/components/PipelineRun/PipelineRunListView/PipelineRunListRow.scss

```diff
@@ -0,0 +1,3 @@
+.git-icon{
+margin:0.2rem 0.3rem 0 0;
```

Comment:
is this missing formatting? The indentation looks off. Also use BEM notation for name. i.e

```
.pipelinerun-list-row {
   &__git-icon {
       // css here
   }
}
```